### PR TITLE
Wl 116 dapps

### PR DESCRIPTION
**Pull request by dev-tronghieu**

### CODE REVIEW

File: apps/dappstore/utils/module.d.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+declare module 'react-native-syntax-highlighter';
+declare module 'react-syntax-highlighter/styles/hljs';
```
Comment (tanlethanh) [inline apps/dappstore/utils/module.d.ts:2]: these modules are unused in this project

File: apps/dappstore/components/layouts/HomeLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+export const HomeLayout: FC<Props> = ({ children }) => {
+	return (
+		<View style={sharedStyles.minScreen}>
+			<View style={sharedStyles.container}>
+				<Header />
+			</View>
+			<ScrollView
+				contentContainerStyle={sharedStyles.contentContainer}
+				showsVerticalScrollIndicator={false}
```
Comment (tanlethanh) [inline apps/dappstore/components/layouts/HomeLayout.tsx:20]: style the scroll indicator instead of hiding it

**Review by tanlethanh — COMMENTED** [review]
I think moving all source code into the `apps/dappstore/src` directory might make the source base cleaner. It will separate code and configurations.

File: apps/dappstore/components/layouts/HomeLayout.tsx (hunk 2 above)
Comment (dev-tronghieu) [reply apps/dappstore/components/layouts/HomeLayout.tsx:20]: I have contacted Ms. Ngoc for the new UI. Will update this soon

**Review by tanlethanh — APPROVED** [review]
@dev-tronghieu I think we could close this PR, resolve confict and move forward. Another update might be in the next PR

**Pull request closed without merging**